Mathlib/MeasureTheory/Integral/IntervalIntegral/ContDiff.lean

```diff
@@ -0,0 +1,111 @@
+/-
+Copyright (c) 2025 Sébastien Gouëzel. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Sébastien Gouëzel
+-/
+import Mathlib.MeasureTheory.Integral.IntervalIntegral.FundThmCalculus
+import Mathlib.Analysis.Calculus.ContDiff.Basic
+
+/-! # Fundamental theorem of calculus for `C^1` functions
+
+We give versions of the second fundamental theorem of calculus under the strong assumption
+that the function is `C^1` on the interval. This is restrictive, but satisfied in many situations.
+-/
+
+noncomputable section
+
+open MeasureTheory Set Filter Function Asymptotics
+
+open scoped Topology ENNReal Interval NNReal
+
+variable {ι 𝕜 E : Type*} [NormedAddCommGroup E] [NormedSpace ℝ E]
+  {f : ℝ → E} {a b : ℝ}
+
+namespace intervalIntegral
+
+variable [CompleteSpace E]
+
+/-- Fundamental theorem of calculus-2: If `f : ℝ → E` is `C^1` on `[a, b]`,
```

Comment:
lower-case after the colon?
```suggestion
/-- Fundamental theorem of calculus-2: if `f : ℝ → E` is `C^1` on `[a, b]`,

Reply:
That's copied from the other file. In English, if there is a full sentence after a colon, it could (should?) start with a capital letter, in theory.

Reply:
If I understand correctly, English generally asks for lower-case after a colon. (Probably, there are some exceptions; I wouldn't be surprised to see some style manual suggest otherwise. English grammar and orthography are... diverse.)

Given this is the same in other files, I won't block on this.